aspnet/mvc/views/forms/sample/final/Controllers/HomeController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.AspNet.Mvc;
+using FormsTH.ViewModels;
```

Comment:
Remove and sort `using`s everywhere. Also sort dependencies in `project.json` files.